packages/cli/src/commands/ts-to-js.js

```diff
@@ -1,6 +1,7 @@
 import { getPaths, convertTsProjectToJs } from '@redwoodjs/internal'
 
 export const command = 'ts-to-js'
+export const aliases = ['tsToJs']
```

Comment:
`tsToJs` --> yuck

`tj` or if strictly sticking with the pattern `ttj`. What say you?

Reply:
`ts2js`

Reply:
I like ts2js too - I also don't think we needto worry too much about this one, because its a very rarely used command. The other ones are commonly used

Reply:
decision --> no alias needed.